src/render/md.jl

```diff
@@ -102,5 +149,69 @@ function writemd(io::IO, m::Meta{:source})
 end
 
 function headermd(io::IO, doc::Metadata, config::Config)
-    print_help(io, config.mdstyle_header, doc.modname)
+    println_mdstyle(io, config.mdstyle_header, doc.modname)
+end
+
+## Docs-specific rendering
+function writemd(io::IO, docs::Docs{:md})
+    println(io, docs.data)
+end
+
+### API-Index ----------------------------------------------------------------------------
+
+type ApiIndex
+    sourcepaths :: Vector{String}
+    modulenames :: Vector{String}
+    entriesmd   :: Vector{EntriesMD}
+
+    ApiIndex() = new(Vector{String}([]), Vector{String}([]), Vector{EntriesMD}([]))
+end
+
+const API_INDEX = ApiIndex()
```

Comment:
Any way we could avoid the mutable global here?

Something like

```julia
index = ApiIndex()
for m in mods
    update_index!(index, save("$(m).md", m))
end
save("index.md", index)
```

as the user interface might be preferable so as to make the `index` state explicit. Would this be possible?